stock_reception_screen/models/stock_reception_screen.py

```diff
@@ -451,10 +451,18 @@ def _validate_current_move(self):
         before checking the next move to process).
         """
         if self.current_move_line_id and self.current_move_id:
-            # We use the 'is_scrap' context key to avoid the generation of a
-            # backorder when validating the move (see _action_done() method in
-            # stock/models/stock_move.py).
-            self.current_move_id.with_context(is_scrap=True)._action_done()
+            moves_todo = self.picking_id.move_ids_without_package.filtered(
```

Comment:
Is there a reason to look only moves without package? Looks dubious to me. If you have 2 remaining moves: one with a package level, one without, if you validate the latter, the picking is set to done and so the move with package level is moved to a backorder. IMO you want `move_lines`

Reply:
Indeed fixed